src/miner.rs

```diff
@@ -152,7 +152,7 @@ pub fn parse_type(raw: &str) -> Result<SqlType> {
             }
             // Map(key, value)
             else if let Some(inner) = extract_inner(raw, "Map") {
-                let (key, value) = inner.split_once(", ").context("invalid map")?;
+                let (key, value) = inner.split_once(",").context("invalid map")?;
```

Comment:
Interesting... 

```sql
CREATE TABLE kek(a UInt8, m Map(String, Float64), d Decimal128(18), e Decimal(36, 18), dt DateTime64(9, 'UTC')) ENGINE = MergeTree ORDER BY a;
```

```sql
SELECT type
FROM system.columns
WHERE table = 'kek'

Query id: b900fd56-a8e2-452c-b5e7-44495748a154

┌─type─────────────────┐
│ UInt8                │
│ Map(String,Float64)  │
│ Decimal(38, 18)      │
│ Decimal(36, 18)      │
│ DateTime64(9, 'UTC') │
└──────────────────────┘
```

I'm going to replace other places with `","` + `trim()` to not rely on implementation details.